wagtail_wordpress_import/prefilters/transform_styles_filter.py

```diff
@@ -62,8 +72,20 @@ def filter_transform_inline_styles_to_tags(html, options=None):
 
             filter[1](soup, tag)
 
-    if TRANSFORM_TAGS_ENABLED:
-        for filter in TRANSFORM_TAGS_MAPPING:
+    transform_html_tags_enabled = getattr(
+        settings,
+        "WAGTAIL_WORDPRESS_IMPORT_TRANSFORM_HTML_TAGS_ENABLED",
+        TRANSFORM_HTML_TAGS_ENABLED,
+    )
```

Comment:
**Question**: There's a bit of duplication here, as we're already gettattring this settings value on line 258 https://github.com/torchbox/wagtail-wordpress-import/pull/49/files#diff-ba7adf08e8b8a91567d4cac5f2059c5cfee02371cfc8aad3b80d29a1046d931aR258. Is this intentional?